src/tests/CMakeLists.txt

```diff
@@ -2,11 +2,10 @@ cmake_minimum_required(VERSION 3.10)
 
 project(unit_tests_antares VERSION 1.0)
 
-# Defining TESTING preprocessor variable at compile time
-# This variable will soon be useless and droped. 
-add_definitions(-DTESTING)
```

Comment:
TESTING variable is now defined with `target_compile_definitions` only for tests that need the variable